content/docs/setup/kubernetes/quick-start-ibm/index.md

```diff
@@ -61,69 +39,95 @@ This guide installs the current release version of Istio.
 1. Install the Helm chart to your cluster:
 
     {{< text bash >}}
-    $ helm install ibm-charts/ibm-istio --name=istio --namespace istio-system
+    $ helm install install/kubernetes/helm/istio --name istio --namespace istio-system --values install/kubernetes/helm/istio/values-istio-demo.yaml
     {{< /text >}}
 
 1. Ensure the pods for the 9 Istio services and the pod for Prometheus are all fully deployed:
 
     {{< text bash >}}
     $ kubectl get pods -n istio-system
-    NAME                                       READY     STATUS      RESTARTS   AGE
-    istio-citadel-748d656b-pj9bw               1/1       Running     0          2m
-    istio-egressgateway-6c65d7c98d-l54kg       1/1       Running     0          2m
-    istio-galley-65cfbc6fd7-bpnqx              1/1       Running     0          2m
-    istio-ingressgateway-f8dd85989-6w6nj       1/1       Running     0          2m
-    istio-pilot-5fd885964b-l4df6               2/2       Running     0          2m
-    istio-policy-56f4f4cbbd-2z2bk              2/2       Running     0          2m
-    istio-sidecar-injector-646655c8cd-rwvsx    1/1       Running     0          2m
-    istio-telemetry-8687d9d745-mwjbf           2/2       Running     0          2m
-    prometheus-55c7c698d6-f4drj                1/1       Running     0          2m
+    NAME                                      READY   STATUS      RESTARTS   AGE
+    grafana-57586c685b-sjs2s                  1/1     Running     0          57m
+    istio-citadel-754b8b478-ggf7s             1/1     Running     0          57m
+    istio-egressgateway-748fb48647-npjgx      1/1     Running     0          57m
+    istio-galley-c66f4f44c-nwb4m              1/1     Running     0          57m
+    istio-ingressgateway-5d444855c8-9ksvn     1/1     Running     0          57m
+    istio-init-crd-10-xcfqb                   0/1     Completed   0          19h
+    istio-init-crd-11-nslct                   0/1     Completed   0          19h
+    istio-init-crd-certmanager-10-2l8hs       0/1     Completed   0          19h
+    istio-pilot-6f6fff9944-twzcp              2/2     Running     0          57m
+    istio-policy-7b6bfcf94d-v6sr7             2/2     Running     2          57m
+    istio-sidecar-injector-6657dd87b9-ccg87   1/1     Running     0          57m
+    istio-telemetry-77d557f66-nsn87           2/2     Running     2          57m
+    istio-tracing-6994cd89bb-gcssk            1/1     Running     0          57m
+    kiali-69d6978b45-5zjzl                    1/1     Running     0          57m
+    prometheus-5488844b5c-vwd9p               1/1     Running     0          57m
+    servicegraph-86b55fc8b8-k87n9             1/1     Running     0          57m
     {{< /text >}}
 
+     _Note:_ The IBM Cloud Kubernetes Service free cluster contains a single worker providing fewer resources, so the `demo` profile was specified. If using a paid cluster of sufficient size, one can remove the --values parameter to use the `values.yaml` values.
```

Comment:
I think this kind of thing is supposed to be a tip: `{{< tip >}} ... {{< /tip >}}`.
Would also probably move it up to right after the command.

Reply:
Done.